### ISSUE TITLE
3Cs setup workflow - add categories?

### ISSUE THREAD

**Issue by michael-wynne-wsu** [MukurtuCMS/Mukurtu-CMS#366]
We have a semi-guided community/protocol workflow. We should include categories in some way.

**Comment by michael-wynne-wsu** [comment]
This could also be addressed by the Tour module, maybe? #708

**Comment by michael-wynne-wsu** [comment]
I'm getting this started. @christa-kavanagh-wsu @joelle-birano-wsu what actions do you think are important to include in a site set-up checklist? We can make different groups of actions.

**Comment by michael-wynne-wsu** [comment]
<img width="1425" height="384" alt="Image" src="https://github.com/user-attachments/assets/c01b4ece-b9e6-4236-bfed-f931e41d675d" />

<img width="1440" height="834" alt="Image" src="https://github.com/user-attachments/assets/014fed7f-5f2b-4840-8ceb-712acf701c03" />

**Comment by christa-kavanagh-wsu** [comment]
I would definitely add Configure a consent popup under Optional. The other optional settings would be good to include as well.

Since we are encouraging folks not to use the UID1 as their everyday account, it might be a good idea to add "Create an admin account" or "Create a user account" on there under Recommended.

**Comment by michael-wynne-wsu** [comment]
Ooh yeah, good catch on the mukurtu manager account.

I don't want to list too many things, otherwise it's just an overwhelming list of options (which are all on the dashboard)

**Comment by christa-kavanagh-wsu** [comment]
That's fair. In that case, would it make more sense to add in some kind of Dashboard tour?

**Comment by michael-wynne-wsu** [comment]
Yes that's on the list too https://github.com/MukurtuCMS/Mukurtu-CMS/issues/708

**Comment by christa-kavanagh-wsu** [comment]
Ah, didn't see that one.

**Comment by michael-wynne-wsu** [comment]
So I think we should keep it pared down to required/recommended, though we could do a "check out these dashboard *sections* for other useful settings"?

**Comment by christa-kavanagh-wsu** [comment]
I like the required/recommended, but if there is a dashboard tour it would probably be best to just direct them to that.
